cmd/root.go

```diff
@@ -57,7 +66,7 @@ var (
 		Example: examples,
 		PreRunE: processBrokerOpts,
 		Run: func(cmd *cobra.Command, args []string) {
-			controllers.StartManager(namespace, metricsAddr, enableLeaderElection, brokerOpts, preparerPluginsPath, debug)
+			controllers.StartManager(namespace, metricsAddr, enableLeaderElection, brokerOpts, preparerPluginsPath, enableLayerCaching, debug)
```

Comment:
@steved This param list is starting to become a bit unwieldy. At some point, either now or if it grows again, we may want to wrap these args in a type and pass that instead.